src/utils/dateHelpers.ts

```diff
@@ -22,7 +24,8 @@ export const beforeNow = (d: string): boolean => {
  * @returns {string} A user-friendly readable date string,
  *   e.g 07/06, 20:09
  */
-export const getPrettyDate = (d: string): string => {
-  const date = moment(d)
-  return date.format('DD/M, HH:mm')
+export const getPrettyDate = (dateString: string): string => {
+  const date = moment(dateString)
+  const prettyDate = date.format('DD/M, HH:mm')
```

Comment:
With 07/06 it is not immediately obvious if we're talking 7th June or 6th July. Can we have the full month name?

Reply:
Oh, that's what you meant!

Reply:
Now it'll show "7th June, HH:MM". I think HHMM is good because there is less ambiguity regarding AM/PM